lang/luajit/Makefile

```diff
@@ -0,0 +1,91 @@
+include $(TOPDIR)/rules.mk
```

Comment:
You should have some kind of copyright disclaimer.
https://forum.lede-project.org/t/packaging-guidelines/65/2